README.md

```diff
@@ -35,30 +35,60 @@ You may run into some issues running the docker commands on Windows:
 
 Other Commands
 --------------
-
-| Docker Command | Description| Host Command |
-|-|-|-|
-| `yarn docker:deps` | Install dependencies for the frontend and backend | `yarn deps` |
-| `yarn docker:start` | Starts the backend and frontend | `yarn start` |
-| `yarn docker:stop` | Stops the backend and frontend |
-| `yarn docker:test` | Runs tests for the frontend and backend |
-| `yarn docker:lint` | Runs the linter for the frontend and backend |
-| `yarn docker:db:migrate` | Run migrations in docker containers | `yarn db:migrate` |
-| `yarn docker:db:migrate:undo` | Undo migrations in docker containers | `yarn db:migrate:undo` |
-| | Install dependencies for the frontend and backend (for local development)  | `yarn deps:local` |
-| | Starts the backend | `yarn server` |
-| | Start the frontend | `yarn client`
-| | Run tests for only the backend | `yarn test`|
-| | Run tests for the backend with coverage and output results to xml files|  `yarn test:ci`|
-| | Run `yarn test:ci` for both the frontend and backend | `yarn test:all`|
-| | Run the linter only for the backend | `yarn lint` |
-| | Run the linter for the the backend with results output to xml files | `yarn lint:ci`|
-| | Run `yarn lint:ci` for both the frontend and backend | `yarn lint:all`|
+| Docker Command | Description| Host Command | Local only Command |
+|-|-|-|-|
+| `yarn docker:deps` | Install dependencies for the frontend and backend | `yarn deps` | `yarn deps:local` |
+| `yarn docker:start` | Starts the backend and frontend | | `yarn start:local` |
+| `yarn docker:stop` | Stops the backend and frontend | |
+| `yarn docker:test` | Runs tests for the frontend and backend | |
+| `yarn docker:lint` | Runs the linter for the frontend and backend | |
+| `yarn docker:db:migrate` | Run migrations in docker containers | `yarn db:migrate` | |
+| `yarn docker:db:migrate:undo` | Undo migrations in docker containers | `yarn db:migrate:undo` | |
+| | Install dependencies for the frontend and backend (for local development)  | `yarn deps:local` | |
+| | Starts the backend | `yarn start` |`yarn server` | |
+| | Start the frontend | `yarn client` | |
+| | Run tests for only the backend | `yarn test`| |
+| | Run tests for the backend with coverage and output results to xml files|  `yarn test:ci`| |
+| | Run `yarn test:ci` for both the frontend and backend | `yarn test:all`| |
+| | Run the linter only for the backend | `yarn lint` | |
+| | Run the linter for the the backend with results output to xml files | `yarn lint:ci`| |
+| | Run `yarn lint:ci` for both the frontend and backend | `yarn lint:all`| |
+
+Integration
+-----------
+
+Linting, unit tests and test coverage analysis are all run automatically on each push
```

Comment:
I'll update this section to mention the security and accessibility reviews once those are in place on the CI pipeline.